### PR TITLE
dev-report: draft for week 22 datakit

### PR DESCRIPTION
**Pull request by avsm**
Signed-off-by: Anil Madhavapeddy <anil@docker.com>

### CODE REVIEW

File: reports/2017-06-04.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Refactor the library to wrap everything under the Datakit namespace ([datakit#583] [@samoht])
+- Simplify the creation of DataKit stores using an Irmin backend ([datakit#584] [@nickbetteridge] [@samoht])
+- Fix ocamldoc ([datakit#585] [@samoht])
+- Various build fixes ([datakit#588] [@talex5]) and jbuilder improvements ([git#218] [@samoht]).
+
+
+## Issues of interest
+
+- [datakit#580] Package `datakit_io' not found - required by `datakit' was opened and closed ([@samoht] [@nickbetteridge])
+- [irmin#451] Documentation improvements had 0 event ()
```
Comment (samoht) [inline reports/2017-06-04.md]: this line looks a bit weird :-)

File: reports/2017-06-04.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Datakit: only the binary is using `datakit_io` ([datakit#582] [@samoht])
+- Refactor the library to wrap everything under the Datakit namespace ([datakit#583] [@samoht])
+- Simplify the creation of DataKit stores using an Irmin backend ([datakit#584] [@nickbetteridge] [@samoht])
+- Fix ocamldoc ([datakit#585] [@samoht])
+- Various build fixes ([datakit#588] [@talex5]) and jbuilder improvements ([git#218] [@samoht]).
+
+
+## Issues of interest
+
+- [datakit#580] Package `datakit_io' not found - required by `datakit' was opened and closed ([@samoht] [@nickbetteridge])
```
Comment (samoht) [inline reports/2017-06-04.md]: this was already covered a bit above, so not sure if it is really of interest.

**Comment by samoht** [comment]
Thanks! Just a few remarks, but as usual very useful and complete.

File: reports/2017-06-04.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+# Weekly DataKit dev report for 2017-05-29 to 2017-06-04 (week 22)
+
+This report covers weekly developments in the [datakit] [irmin] [git] [9p] repositories.
+
+This week saw a significant simplication in the GitHub bridge is deployed, as it can now run without a server and use a local Git repository directly ([datakit#577] [@talex5] [@samoht]).  This in turn simplifies the deployment of DataKit as part of various CI services use it now, such as the [LinuxKit](https://github.com/linuxkit/linuxkit) Moby project.
```
Comment (talex5) [inline reports/2017-06-04.md]: "in *how* the"

File: reports/2017-06-04.md (hunk 1 above)
Comment (avsm) [inline reports/2017-06-04.md]: fixed in 2bc47fe09a28d1e41b5d712dede1b98ec2f6876e with more info

File: reports/2017-06-04.md (hunk 2 above)
Comment (avsm) [inline reports/2017-06-04.md]: fixed in 2bc47fe09a28d1e41b5d712dede1b98ec2f6876e

File: reports/2017-06-04.md (hunk 3 above)
Comment (avsm) [inline reports/2017-06-04.md]: also fixed in 2bc47fe09a28d1e41b5d712dede1b98ec2f6876e

**Comment by avsm** [comment]
also noted slack channel as suggested by @samoht in Slack

**Comment by samoht** [comment]
Thanks!

**Pull request merged**